deployment/terraform/step-functions/etl.json.tmpl

```diff
@@ -0,0 +1,85 @@
+{
+	"StartAt": "Step1",
+	"TimeoutSeconds": 10800,
+	"States": {
+		"Step1": {
+			"Type": "Task",
+			"Resource": "arn:aws:states:::batch:submitJob.sync",
+			"Parameters": {
+				"JobName": "DropSourceSchemas",
```

Comment:
```suggestion
				"JobName": "PullFromKfDataservice",
```

Reply:
I know the following change suggestions don't matter too much since these steps are just placeholders, but it would nice if we could make the placeholders more related/meaningful to the FHIR ETL